### DISCUSSION TITLE
LInux CentOS 7.9  按照这个教程没法正确安装rime

### DISCUSSION THREAD

**Discussion by mrdashuai** [rime/home#1256] [Q&A]
进入 ibus-rime 之后用make命令出现以下错误，并且全网未见有针对该问题的解决办法。

CMake Error at CMakeLists.txt:25 (find_package):
  By not providing "FindRime.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Rime", but
  CMake did not find one.

  Could not find a package configuration file provided by "Rime" with any of
  the following names:

    RimeConfig.cmake
    rime-config.cmake

  Add the installation prefix of "Rime" to CMAKE_PREFIX_PATH or set
  "Rime_DIR" to a directory containing one of the above files.  If "Rime"
  provides a separate development package or SDK, be sure it has been
  installed.

-- Configuring incomplete, errors occurred!
See also "/usr/src/ibus-rime/build/CMakeFiles/CMakeOutput.log".
make: *** [ibus-engine-rime] Error 1

**Comment by MokOopsing** [comment]
从源码安装请参考这个 https://github.com/rime/home/wiki/RimeWithIBus#manual-installation

如果还不可以的话，请把CMakeOutput.log贴出来看看

**Comment by mrdashuai** [comment]
我就是参考这个安装的，已经花了两天时间用来解决安装过程中遇到的问题了，不过还是有好多问题没法解决，造成安装无法完成。现在放弃了。试过RedHat 7/8/9 和 CentOS 7.2/7.9 都无法按照教程完成安装。等其他人出什么博客知乎教程时候，我再来试吧。
现是用了 sudo yum install ibus-table-wubi 这个五笔输入法，也挺好用的。暂时将就着吧。

**Comment by mrdashuai** [comment]
我认为从来没有人成功安装过Linux版本的Rime，无论是谁看到这个信息，碰巧你却成功安装了Linux Rime，麻烦发截图给我吧，感谢。
邮箱：mrdashuai@outlook.com

**Reply by MokOopsing** [reply]
我之前在fedora上安装ibus-rime是没什么问题的，不过我最新换成了uos/deepin，截图什么的就不方便了，不过我觉得别人装成功了的截图也不会对你有帮助的，自己提供日志才会有用

**Reply by mrdashuai** [reply]
在Linux上安装这个RIME有如自己在写一个五笔代码，不想再折腾了，等你们谁提供一个完整的安装教程时候，我再试。

**Reply by MokOopsing** [reply]
只能你自己来了，你不用不是任何人的损失

**Reply by mrdashuai** [reply]
兄弟，我不用也不是我的损失呀。你这话说得，真操蛋。
